Px.Utils/Models/Data/DataValue/DecimalDataValue.cs

```diff
@@ -11,17 +11,26 @@ namespace PxUtils.Models.Data.DataValue
         /// Numeric value of the data point when the type is <see cref="DataValueType.Exists"/>.
         /// Otherwise the value is undefined and using it will cause undefined behavior.
         /// </summary>
-        public readonly decimal UnsafeValue;
+        public readonly decimal UnsafeValue
+        {
+            get { return _unsafeValue; }
```

Comment:
No need to use private flields here. public readonly decimal UnsafeValue { get; } is enough

Reply:
Yeah just noticed that being used in another branch